docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,212 @@
+
+- Shared controls - Responsibilities that vary depending on the service model.
+
+  Infrastructure as a Service (IaaS), Platform as a Service (PaaS), or Software as a Service (SaaS),  such as cloud email:
+
+  - IaaS (Infrastructure-as-a-Service): A hybrid approach where organizations can manage some of their data and applications on-premise while relying on cloud providers to manage servers, hardware, networking, virtualization, and storage needs.
+
+  - PaaS (Platform-as-a-Service): Provides organizations the ability to streamline their application development and delivery by providing a custom application framework that automatically manages operating systems, software updates, storage, and supporting infrastructure in the cloud.
+
+  - SaaS (Software-as-a-Service): Cloud-based software hosted online and typically available on a subscription basis. Third-party providers manage all potential technical issues, such as data, middleware, servers, and storage, minimizing IT resource expenditures and streamlining maintenance and support functions.
+
+For inherited controls, you can rely on AWS to provide the security of the cloud itself. For customer-specific security tasks, the apps and configurations specific to your unique circumstances are running on the AWS platform. For example, planning on managing the security of your instances, including user access controls (IAM), managing patches in your operating systems and which firewall ports are open/closed in your instances.
+
+### Auditing Cloud Environments with Chef
+
+Chef scans systems across all environments (Dev, Pre-Production, Production), all systems (Cloud, Kubernetes, VMs, Containers, Windows, Linux), and all clouds (AWS, Azure, Google, Alibaba).
+
+You have access to the Chef InSpec resources library, including benchmarks and profiles created by the Center for Internet Security (CIS). The CIS benchmark turns security best practices into specific, actionable controls that can run against your cloud systems. In addition, the Chef CIS benchmarks prevent the misconfiguration in the first place itself. Thus, fixing the misconfigurations in the Infrastructure As Code (IaC) used to create the resources.
+
+The CIS/DISA STIG certified profiles are available for Amazon Web Services, Microsoft Azure, GCP, Alibaba Cloud, and Google Cloud/Compute platforms. These profiles provide insights about services like identity management, security groups, and storage buckets that are implemented securely. With Chef InSpec's easy-to-code language and built-in profiles and resources, you can address your organization's concerns. And with Chef Infra, you can take corrective actions based on the insights derived.
+
+Policy as code with provisioning tools like Terraform merges infrastructure-as-code and compliance-as-code into a single workflow. The Chef InSpec can be used with *Terraform* in two different ways to confirm compliance:
```

Comment:
```suggestion
Policy as code with provisioning tools like Terraform merges infrastructure-as-code and compliance-as-code into a single workflow. Chef InSpec can be used with *Terraform* in two different ways to confirm compliance:
```

Reply:
I guess, the article 'The' is required. Isn't it as we are calling it 2nd/ 3rd time in the writeup - same page?